x-pack/test/apm_api_integration/tests/dependencies/top_operations.spec.ts

```diff
@@ -52,6 +52,7 @@ export default function ApiTest({ getService }: FtrProviderContext) {
             environment,
             kuery,
             dependencyName,
+            searchServiceDestinationMetrics: false,
```

Comment:
Should we have two API tests that ensure we get the same response back regardless if `searchServiceDestinationMetrics` is true or false?